### DISCUSSION TITLE
Error OFFSETPART2 in DB and RCC flash do NOT match. Stopping here ...

### DISCUSSION THREAD

**Discussion by rdcrampton** [Mr-MIBonk/M.I.B._More-Incredible-Bash#330] [Q&A]
Hello all. I am new to the community after recently purchasing a 2015 Škoda Octavia RS. Current SW Train version is MHIG_RW_SK_P1537_1 and software version 14.19.0_H15.17.1518_HIGH_EU. I am attempting to enable CarPlay and have successfully created the SD card with MIB working and GEM installed. When I attempt the AIO patch it gets almost all the way through and then throws error Unet/mmx/ts/sda0/apps/sbin/flashlock] info: Root-Block at Offset. (9)!
ifs-root-stage2.ifs offset: 0x00bc0000
/net/mmx/ts/sda0/apps/flash[397]: [: -: missing second argument ifs-root-stage2.ifs size: E45ECCOO
I Error: OFFSETPART2 in DB and RCC flash, do-NOT match.
Stopping here.

What am I missing? Do I need to update the firmware before attempting the patch? I’m using a 64GB sd card formatted exfat on a windows machine. Please excuse my ignorance if I have missed something obvious.

![IMG_6994](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/17489362/f33a9898-46b7-4b5a-9113-a5a764889fcf)
![IMG_6997](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/17489362/63f39a89-57aa-45e5-8f04-5e2d11db0cbe)

**Comment by rdcrampton** [comment]
Ok so I now understand that I need to update the MHIG SW Train to the latest version and found my way to the relevant file on the repository MHIG_EU_SK_K1552 MU0411. My issue now is the region conflict when trying to update. Can I switch the region in GEM or force ignore?

**Reply by kamilcwalk94-blip** [reply]
How did you update it? i have MHIG_EU_AU_K1544 MU0397 and have same error

**Comment by rdcrampton** [comment]
Ok I have figured out how to convert to EU region. Now to attempt to flash after updating the Train and enable CarPlay.

**Comment by aledepaoligampel** [comment]
Hi @rdcrampton ! can you solve your problem?
